internal/git/repository.go

```diff
@@ -158,7 +158,7 @@ func (r *Repository) Setup() {
 	})
 
 	r.statusShortOnce = sync.OnceValues(func() ([]string, error) {
-		return r.Git.CmdLines(cmdStatusShort)
+		return r.Git.CmdLinesNoTrim(cmdStatusShort)
```

Comment:
Unrelated to this PR, note that `Git.CmdLines` trimming leading space would create issues when the first file returned by `git status --short` is unstaged, as the leading space is, in that case, meaningful.